ELSpace/Screens/Hub/Activity/Form/ActivityFormViewController.swift

```diff
@@ -0,0 +1,189 @@
+import UIKit
+import RxSwift
+
+class ActivityFormViewController: UIViewController, ActivityFormViewControlling, UITextFieldDelegate {
+
+    init(viewModel: ActivityFormViewInputModeling & ActivityFormViewOutputModeling) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func loadView() {
+        view = ActivityFormView()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureSubviews()
+        setInitialState()
+        setupInputBindings()
+        setupOutputBindings()
+    }
+
+    // MARK: - ActivityFormViewControlling
+
+    var type: AnyObserver<ActivityType> {
+        return viewModel.type
+    }
+
+    var date: AnyObserver<Date> {
+        return viewModel.date
+    }
+
+    var form: Observable<ActivityForm> {
+        return viewModel.form
+    }
+
+    // MARK: - Privates
+
+    private let viewModel: ActivityFormViewInputModeling & ActivityFormViewOutputModeling
+
+    private var editingTextField: UITextField? {
+        didSet {
+            if let inactiveTextView = oldValue?.superview as? TextView {
+                inactiveTextView.separatorLine.backgroundColor = titleColorForState(false)
+                inactiveTextView.endEditing(true)
+            }
+            if let activeTextView = editingTextField?.superview as? TextView {
+                activeTextView.separatorLine.backgroundColor = titleColorForState(true)
+            }
+        }
+    }
+
+    private let disposeBag = DisposeBag()
+
+    private var activityFormView: ActivityFormView! {
+        return view as? ActivityFormView
+    }
+
+    private func configureSubviews() {
+        activityFormView.dateTextView.textField.isEnabled = false
+        activityFormView.projectTextView.textField.delegate = self
+        activityFormView.pickerView.showsSelectionIndicator = true
+        activityFormView.pickerView.isHidden = true
+        activityFormView.hoursTextView.textField.autocorrectionType = .no
+        activityFormView.hoursTextView.textField.keyboardType = .decimalPad
+        activityFormView.hoursTextView.textField.delegate = self
+        activityFormView.commentTextView.textField.autocorrectionType = .no
+        activityFormView.commentTextView.textField.returnKeyType = .done
+        activityFormView.commentTextView.textField.delegate = self
+    }
+
+    private func setInitialState() {
+        activityFormView.dateTextView.separatorLine.backgroundColor = titleColorForState(false)
```

Comment:
Look at previous comment

Reply:
I moved setup of static properties to view but dynamic ones left in viewController for keeping the logic in the same class